src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,81 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private int indexForNexElem = 0;
+    private int size;
+    private K[] keyStorage;
+    private V[] valueStorage;
+
+    public StorageImpl() {
+        size = 0;
+        keyStorage = (K[]) new Object[MAX_SIZE];
+        valueStorage = (V[]) new Object[MAX_SIZE];
+    }
+
     @Override
     public void put(K key, V value) {
+
```

Comment:
It's better to replace this checking in the beginning of the metho
```suggestion
if (size == MAX_SIZE) {
                throw new RuntimeException("The storage is full");
            }
```